RingPublishingTracking/src/main/java/com/ringpublishing/tracking/data/TrackingIdentifier.kt

```diff
@@ -11,7 +11,14 @@ import java.util.Date
 /**
  * Tracking identifier
  *
- * @param identifier from me call
- * @param expirationDate of identifier
+ * @param userIdentifier base tracking identifier
+ * @param artemisIdentifier artemis tracking identifier
  */
-data class TrackingIdentifier(val identifier: String, val expirationDate: Date)
+data class TrackingIdentifier(
+    val userIdentifier: UserIdentifier,
+    val artemisIdentifier: ArtemisIdentifier
```

Comment:
Jeśli całość będzie ok, to oczywiście zaktualizuje interfejs publiczny względem iOS